x-pack/platform/plugins/shared/streams/server/lib/streams/assets/query/query_client.ts

```diff
@@ -229,7 +235,7 @@ export class QueryClient {
           query: esqlQuery,
         },
         enabled: true,
-        tags: ['streams'],
+        tags: ['streams', stream],
```

Comment:
Added the stream name as tag to help differentiate a query that was imported into different targets

Reply:
That will help for sure, but the end goal is to have these rules hidden from the stack management alerts page